app/components/joyride-foundation.js

```diff
@@ -43,6 +54,10 @@ export default Ember.Component.extend({
         function showJoyrideOverlay(){
           Ember.$(".joyride-view-background").show();
         }
+
+        function setSeenTour(){
+          _this.set("session.seenTour", _this.get("session.seenTour") + "," + _this.get("currentUserId"));
```

Comment:
Ah ok, I see what you mean. That's quite an edge case! I think if we care enough about this we should persist the 'seen-joyride' bool to the database and return it with the user profile. As you point out, the current solution doesn't handle other browsers. My pref would be to ignore this case for now as it really only pops up in testing.